src/WebJobs.Script/Description/DotNet/DotNetFunctionDescriptionProvider.cs

```diff
@@ -174,6 +174,7 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
 
             // We were unable to compile the function to get its signature,
             // setup the descriptor with the default parameters
+            methodAttributes.Clear();
```

Comment:
This fixes the "multiple attributes" issue we've been seeing. Above we were adding a [NoAutomaticTrigger] attribute to the methodAttribute collection as part of compilation, but when that fails, the collection still contains that attribute, and the following method adds another, causing the core SDK to throw.